### ISSUE TITLE
website down https://openjarvis.com please update

### ISSUE THREAD

**Issue by rajkiranjeet** [alexylem/jarvis#841]
### Description

Français and English are allowed.
Detailed description of your problem.
In case of feature request, please provide justification & example of usage.

### Logs
```shell
<PASTE HERE> full content of the terminal with Jarvis in troubleshooting mode
(keep triple quotes above and below to preserve formatting) 
```
Alternatively attach a terminal session recording using asciinema:
```shell
sudo apt-get install -y asciinema && asciinema rec # start session recording
./jarvish.sh -nv # start jarvis directly in troubleshooting mode
Ctrl+D # stop recording
y # yes to upload and get URL to paste here
```

**Comment by Pofilo** [comment]
You can temporary access the website in cache here: [https://web.archive.org/web/20180424003049/https://openjarvis.com/](https://web.archive.org/web/20180424003049/https://openjarvis.com/).

**Comment by alexylem** [comment]
Looking at it

**Comment by alexylem** [comment]
Webserver ok, htaccess ok, quota ok, Database ok...
Just put a maintenance page
Further investigating...

**Comment by alexylem** [comment]
Seems related to PHP simplest page without .htaccess fails, contacted OVH

**Comment by rajkiranjeet** [comment]
one more question i see jarvis working ok on lots of youtube video but when i test its not recognize trigger word.any chance to  get English version pre compiled raspberry pi3 firmware

**Comment by alexylem** [comment]
Site back online.
Found out jarvis main folder was 777 instead of 755... apparently OVH has added some security checks, which is good.

**Comment by Oliv4945** [comment]
Thanks !

**Comment by cganiddle** [comment]
Problem still there it seems.

**Comment by alexylem** [comment]
Yep thanks, fixed again. I don't understand why something keeps changing the permissions of the folder.... I leave it open to find the root cause

**Comment by alexylem** [comment]
Didn't find anything that can be changing folder permissions...
I see two options:
1. change ftp password... that is a quick win and will automatically any forgotten / hacker process screwing up the filesystem
2. update drupal, there are pending security updates, that will help but will need outage of the site
